torch/distributed/_composable/fsdp/_fsdp_param.py

```diff
@@ -428,7 +428,6 @@ def unsafe_setattr_param(
 ) -> None:
     if getattr(module.__setattr__, "__func__", None) is nn.Module.__setattr__:
         module._parameters[param_name] = param
-        super(nn.Module, module).__setattr__(param_name, param)
```

Comment:
Curious what purpose this originally served since we already did `module._parameters[param_name] = param`.

Reply:
I think it did not serve any purpose and was a mistake 😅